### PR TITLE
Feature/split out admin styles

### PR DESCRIPTION
**Pull request by pzi**
Clearly separate admin styles from public/member styles in preparation for them to be moved into its own gem.

Bonus: Fix for #93

### CODE REVIEW

File: app/assets/stylesheets/admin/admin.sass [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// This file contains all Admin styles.
+// -----------------------------------------------------------------------------
+
+@import "normalize-rails"
+@import "bourbon"
+
+/*!
+ * Author:       The Frontier Group
+ * Author URI:   http://www.thefrontiergroup.com.au/
+ */
```
Comment (jordanmaguire) [inline app/assets/stylesheets/admin/admin.sass]: Is there any reason we have two comment blocks? Can they be combined in some way?

**Comment by jordanmaguire** [comment]
Seems aiight to me :+1:

File: app/assets/stylesheets/admin/admin.sass (hunk 1 above)
Comment (pzi) [inline app/assets/stylesheets/admin/admin.sass]: Yes (1 and No (2

they have different purposes and behaviour:

Comment on L1 doesn't get output (it's silent) and describes the file / its contents
Comment on L8 is a "PSA" and will appear in the compiled stylesheet – the `/*!` causes that

>This is useful for adding Copyright notices to your generated CSS.

See [Sass Reference](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments)

File: app/assets/stylesheets/admin/global/_base.sass [hunk 2]
```diff
@@ -16,6 +21,18 @@ div.container
   span
     @extend %visually-hidden
 
+thead, h1, h2, h3, h4, h5, h6
+  color: $heading-color
+
+thead, h1, h2, h3
+  font-family: $heading-fonts
+
+h1, h2, h3, h4, h5, h6
```
Comment (jgunnink) [inline app/assets/stylesheets/admin/global/_base.sass]: Any reason these are different to the bugfix in this file? https://github.com/thefrontiergroup/cciwa-web/blob/bugfix/fix-text-in-inputs/app/assets/stylesheets/admin/global/_base.sass

Comment (pzi) [inline app/assets/stylesheets/admin/global/_base.sass]: Ultimately, there is no difference, the compiled output will be the same.

The difference here is that I just shuffled files around and didn't spend any time formatting the actual code.
It's just a visual thing and in CCIWA I had a [Sass Linter](https://github.com/sasstools/sass-lint) running that was complaining about [the format](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/single-line-per-selector.md).

My goal is that eventually all files using or in Rails template are formatted the same. Can't spend that much time updating it all though. So whenever I work on a piece and have extra time, I will convert the files I am touching.

In this case, all the CSS files will be changed again later on when we have the gem, so I didn't spend any time formatting yet.

Comment (jgunnink) [inline app/assets/stylesheets/admin/global/_base.sass]: Cool man. I have removed the link from my comment as it's a link to a non-public repo.

File: app/assets/stylesheets/admin/admin.sass (hunk 1 above)
Comment (pzi) [inline app/assets/stylesheets/admin/admin.sass]: Actually, to be more more accurate:

`//` = internal comment that will never appear in any form of compiled css
`/*` = comment will appear in compiled css but will get stripped out by minifiers
`/*!` = comment will appear in compiled css but will **NOT** get stripped out by minifiers

**Pull request merged**